packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -37,6 +46,15 @@ export class Telemetry {
       Telemetry.client.addTelemetryProcessor(sanitizeEnvelope);
     }
   }
+
+  static isCI(): boolean {
```

Comment:
Shouldn't we just pass --no-telemetry? (Or perhaps just tag the CI builds?)

Reply:
I can see we don't want telemetry from our CI, but shouldn't we get metrics of customers CI like crashes, durations to pin down slowness?

Reply:
talked offline - we'll want to figure out cost for a month and the we can reevaluate changing the "disable" to "log and tag as CI"